mathics/doc/gather.py

```diff
@@ -138,7 +139,7 @@ def gather_sections(chapter, module, builtins_by_module, section_class=None) ->
     # converting the entries into `set`s.
     #
     visited = set()
-    for symbol_instance in builtins_by_module[module.__name__]:
+    for symbol_instance in builtins_by_module.get(module.__name__,[]):
```

Comment:
Actually, the main change in this file is this line. To move the other functions is just to improve the organization of the code.